crates/orchestrator/src/config.rs

```diff
@@ -491,10 +498,20 @@ pub struct NetworkConfigFile<KEY: SignatureKey> {
     /// random builder configuration
     #[serde(default)]
     pub random_builder: Option<RandomBuilderConfig>,
+    /// The list of public keys that are allowed to connect to the orchestrator
+    #[serde(default)]
+    pub public_keys: HashSet<KEY>,
+    /// Whether or not to disable registration verification.
+    #[serde(default)]
+    pub disable_registration_verification: bool,
 }
 
 impl<K: SignatureKey> From<NetworkConfigFile<K>> for NetworkConfig<K> {
     fn from(val: NetworkConfigFile<K>) -> Self {
+        if val.disable_registration_verification {
+            tracing::error!("REGISTRATION VERIFICATION IS TURNED OFF");
+        }
+
```

Comment:
I think this should probably go in `run_orchestrator` in `orchestrator/src/lib.rs`, but that's pretty minor